### PR TITLE
javascript/javascript_in_the_real_world/linting.md: added info about the eslint and prettier conflict when used together and a fix

### PR DESCRIPTION
**Pull request by r0hitm**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

- The lesson on linting discusses both `eslint` and Prettier but does not states that using them together causes conflicts. This
PR clarifies that.
- And provies a way to let user use them together by installing `eslint-config-prettier`. [Link to install instructions](https://github.com/prettier/eslint-config-prettier#installation)

**3. Additional Information:**

[Link to Discord suggestions-bugs post](https://discord.com/channels/505093832157691914/1023984062605578351)

### CODE REVIEW

**Comment by Lofty-Brambles** [comment]
Would it be pertinent to talk about `esint-plugin-prettier` and link to the [prettier docs page](https://prettier.io/docs/en/integrating-with-linters.html) where it talks about them as well?

**Comment by r0hitm** [comment]
> Would it be pertinent to talk about `esint-plugin-prettier` and link to the [prettier docs page](https://prettier.io/docs/en/integrating-with-linters.html) where it talks about them as well?

The [Notes](https://prettier.io/docs/en/integrating-with-linters.html#notes) section of the Prettier docs say:
> When searching for both Prettier and your linter on the Internet you’ll probably find more related projects. These are **generally not recommended**, but can be useful in certain circumstances.
> These plugins were especially useful when Prettier was new.

And it also lists the downside of using the `eslint-plugin-prettier`.

**Comment by zachmmeyer** [comment]
In agreement with the Prettier docs, I do not believe this is the best solution. I would maybe link to the doc page that @Lofty-Brambles mentioned and let people figure out what they would want to do while being informed. That way, if they want to go ahead and install that plugin after learning about the downsides, they can. Or they can take other steps to have less conflicts.

**Comment by r0hitm** [comment]
@zachmmeyer I agree. That would be a better way of doing it. How do I go about editing the commit? If I edit the commit it will not be reflected in this PR (will it?), Do I have to create a new PR?

File: javascript/javascript_in_the_real_world/linting.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ Prettier is _awesome_. It is similar to a linter, but serves a slightly differen
 
 Using prettier makes coding faster and easier! You don't have to worry about nailing things like indentation, or remembering every semi-colon because prettier will take care of those details for you.
 
+### Note on using Eslint and Prettier together
+
+Using eslint and prettier together causes conflicts. To fix that follow the instructions to install [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier#installation). It turns off all eslint rules that are unnecessary or might conflict with Prettier. Doing just this is enough to resolev the conflict and get them both working smoothly with one another.
```
Comment (zachmmeyer) [inline javascript/javascript_in_the_real_world/linting.md:41]:
```suggestion
Using ESLint and Prettier together causes conflicts. To fix that follow the instructions to install [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier#installation). It turns off all ESLint rules that are unnecessary or might conflict with Prettier. Doing just this is enough to resolve the conflict and get them both working smoothly with one another.
```
There's a typo, as well as fixing capitalization on Prettier and ESLint.

File: javascript/javascript_in_the_real_world/linting.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ Prettier is _awesome_. It is similar to a linter, but serves a slightly differen
...
+Another way to address the conflict is to use `eslint-plugin-prettier`. It lets you run prettier as if it was a linter rule. However, doing this is **not recommended**. You can learn more about it [here](https://prettier.io/docs/en/integrating-with-linters.html#notes)
```
Comment (zachmmeyer) [inline javascript/javascript_in_the_real_world/linting.md:42]:
```suggestion
Another way to address the conflict is to use `eslint-plugin-prettier`. It lets you run Prettier as if it were a rule in ESLint. However, doing this is **not recommended**. You can learn more about it [here](https://prettier.io/docs/en/integrating-with-linters.html#notes)
```
Fixing capitalization and changing up sentence structure a bit.

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
Thanks for making those changes. I have a few requests on the final contribution.

**Pull request merged**